generate-err-logs.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
+
+
+main() {
+    echo "Generating error logs" >&2
+    generate_logs
+    echo "Generated logs sucessfully" >&2
+}
+
+generate_logs() {
+    local pod_definitions_file="failed-pods-definitions.yaml"
+    local pod_logs_file="failed-pods-logs.log"
+    local event_messages_file="failed-deployment-event-log.log"
+
+    cat /dev/null > "$pod_logs_file"
+    cat /dev/null > "$pod_definitions_file"
+    cat /dev/null > "$event_messages_file"
```

Comment:
Maybe this will be more straightforward: 
```suggestion
    rm -f "$pod_logs_file" "$pod_definitions_file" "$event_messages_file"
```
Maybe can put them all inside a single directory and then we can delete the directory instead.
Also, I think we should add those files or the directory to .gitignore.

Reply:
yup, that actually makes much more sense. 
added